api/Models/DefaultDbContext.cs

```diff
@@ -1,21 +1,51 @@
+using aspnetCoreReactTemplate.Extensions;
+using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore;
 
 namespace aspnetCoreReactTemplate.Models
 {
-    public class DefaultDbContext : IdentityDbContext<ApplicationUser>
+  public class DefaultDbContext : IdentityDbContext<ApplicationUser>
+  {
+    public DefaultDbContext(DbContextOptions<DefaultDbContext> options)
+      : base(options)
     {
-        public DefaultDbContext(DbContextOptions<DefaultDbContext> options)
-        : base(options)
+    }
+
+    public DbSet<ApplicationUser> ApplicationUsers { get; set; }
+    public DbSet<Contact> Contacts { get; set; }
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+      base.OnModelCreating(modelBuilder);
+
+      // Replace table names
+      foreach(var entity in modelBuilder.Model.GetEntityTypes())
+      {
+
+        entity.Relational().TableName = entity.Relational().TableName.Replace("AspNet", "").ToSnakeCase();
+
+        // Replace column names
```

Comment:
Can you make this  comment (and others in this file) note that we are doing PascalCase > camelCase translation.  It's fairly obvious but I think it would be good to have the comments spell it out.

Reply:
Done. Also added some examples to some of the comments.